telemetry/istio/src/main/scala/io/buoyant/telemetry/istio/IstioTelemeterInitializer.scala

```diff
@@ -0,0 +1,53 @@
+package io.buoyant.telemetry.istio
+
+import com.fasterxml.jackson.annotation.JsonIgnore
+import com.twitter.finagle.{Address, Name, Stack}
+import com.twitter.finagle.buoyant.H2
+import com.twitter.finagle.util.DefaultTimer
+import com.twitter.logging.Logger
+import io.buoyant.telemetry.{TelemeterConfig, TelemeterInitializer}
+
+class IstioTelemeterInitializer extends TelemeterInitializer {
+  type Config = IstioConfig
+  val configClass = classOf[IstioConfig]
+  override val configId = "io.l5d.istio"
+}
+
+object IstioTelemeterInitializer extends IstioTelemeterInitializer
+
+private[istio] object IstioConfig {
+  val DefaultHostname = "127.0.0.1"
+  val DefaultPort = 9091
+}
+
+private[istio] case class IstioConfig(
+  hostname: Option[String],
+  port: Option[Int]
+) extends TelemeterConfig {
+  import IstioConfig._
+
+  @JsonIgnore override val experimentalRequired = true
+
+  @JsonIgnore private[this] val log = Logger.get("io.l5d.istio")
```

Comment:
Might just use `Logger()` here since io.l5d.istio could refer to a number of different things